src/genie/libs/parser/iosxe/show_snmp.py

```diff
@@ -80,6 +81,329 @@ def cli(self, output=None):
         return ret_dict
 
 
+
+# ==========================
+# Schema for 'show snmp'
+# ==========================
+class ShowSnmpSchema(MetaParser):
+
+    ''' Schema for "show snmp" '''
+
+    schema = {
+        "chassis": str,
+        "contact": str,
+        Optional("location"): Optional(str),
```

Comment:
`Optional()` only applies to the key, it's not needed for the value types as well.
```suggestion
        Optional("location"): str,
```